dali/python/nvidia/dali/ops.py

```diff
@@ -721,7 +729,7 @@ class PythonFunction(PythonFunctionBase):
 
     @staticmethod
     def current_stream():
-        """Get DALI's current CUDA stream."""
+        """Gets DALI’s current CUDA stream."""
```

Comment:
```suggestion
        """Gets DALI's current CUDA stream."""
```
I don't know what character it was, but it was scary to see one in a source code file....

Reply:
Done